README.md

```diff
@@ -59,9 +59,13 @@ class TestModel(spanner_orm.Model):
   value_index = spanner_orm.Index(['value'])
 
   # To indicate that there is a foreign key relationship from this table to
-  # another one, use a Relationship. This has no impact on the representation
-  # of the table inside Spanner
-  fake_relationship = spanner_orm.Relationship(
```

Comment:
hm unclear if this makes it a breaking change. It doesn't affect someone using `Relationship` currently, but it change the behavior of that class going forward (say if you add a new model / new relationship). Although, you could also argue that this was a bug in the library that this PR fixes..

Reply:
Yeah I think this has to be the path forward. This was one of the first things that jumped out at me when I first found the library and read the `README`. When I saw, "fake_relationship` and "no impact to the representation of the table inside Spanner", it jumped out at me as a major limitation since these relationships/constraints must exist in a real-world production-level Spanner instance.